src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -101,6 +109,7 @@ public KubernetesCloud(String name, List<? extends PodTemplate> templates, Strin
 
         this.serverUrl = serverUrl;
         this.namespace = namespace;
+
```

Comment:
ok, then i guess the getSystemNamespace() call wouldn't belong here anyway.  thanks!